samples/nrf9160/location/nrf9160dk_with_nrf7002ek.overlay

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+/* Use high drive mode on SPI pins that handle communication with nRF7002
+ * so that SCK frequency > 4 MHz can be used. */
+ &spi3_default {
+	group1 {
+		nordic,drive-mode = <NRF_DRIVE_H0H1>;
+	};
+};
+
+/* Disabled because of conflicts on P0.00 and P0.01 - Arduino pins D0 and D1
+ * (iovdd-ctrl-gpios and bucken-gpios in nrf7002_ek, respectively). */
+&uart1 {
+	status = "disabled";
+};
+
+/* TODO:
+/ {
+	chosen {
+		ncs,location-wifi = &nrf7002_ek;
+	};
+};
+*/
```

Comment:
Is this not enough to use `DEVICE_DT_GET` ?

Reply:
I thought so as well but no. 

For `nrf7002_ek`:

-- Found BOARD.dts: /home/jani/code_wa/ncs/zephyr/boards/arm/nrf9160dk_nrf9160/nrf9160dk_nrf9160_ns.dts
-- Found devicetree overlay: /home/jani/code_wa/ncs/nrf/boards/shields/nrf7002_ek/nrf7002_ek.overlay
-- Found devicetree overlay: nrf9160dk_with_nrf7002ek.overlay
devicetree error: /chosen: undefined node label 'nrf7002_ek'

With `nrf7002` it is going further in compilation but when linking:
/home/jani/zephyr-sdk-0.14.1/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/10.3.0/../../../../arm-zephyr-eabi/bin/ld.bfd: modules/nrf/lib/location/lib..__nrf__lib__location.a(method_wifi.c.obj): in function `method_wifi_init':
/home/jani/code_wa/ncs/nrf/lib/location/method_wifi.c:297: undefined reference to `__device_dts_ord_111'

Reply:
The label `nrf7002_ek` is under `arduino_spi`, maybe we should use the same hierarchy?

Reply:
Seems that generated zephyr.dts is not having label `nrf7002_ek ` but it is having a label `nrf7002`.
...and that is ending to linker error as shown above.